.github/workflows/build-dumb-pypi.yml

```diff
@@ -0,0 +1,177 @@
+name: Build dumb PyPI on GH pages
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      ansible-base-slug:
+        default: ansible/ansible
+        description: The GitHub repo slug for pulling ansible-base from
+        required: true
+      ansible-base-ref:
+        default: devel
+        description: The GitHub repo ref for pulling ansible-base from
+        required: true
+      ansible-build-data-slug:
+        default: ansible-community/ansible-build-data
+        description: The GitHub repo slug for pulling ansible build data from
+        required: true
+      ansible-build-data-ref:
+        default: main
+        description: The GitHub repo ref for pulling ansible build data from
+        required: true
```

Comment:
@abadger look at this. This is experimental and is supposed to add a button allowing you to trigger a workflow from the GitHub UI. I haven't tested it but it should appear after the merge.

Ref: https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
@abadger this specific piece of setup could be used to automate generating (and potentially publishing) the stable releases to PyPI. Do you have any opinion on this? IIRC the build happens by manual runs locally, right?

Reply:
I do want to have a push button to make a release eventually.  There's one piece of coordination involved in making a release right now, though, updating the docsite.  Possibilities:

* Make the build one push button and publish a second one.  That way the release manager can alert the docs team in between.
* Ask the docs team if they can update the docs after we've published.  Now that the docs build is back down to ~ ten minutes I think they might be okay with that.

Note that building and publishing includes saving information about the build/release to the ansible-build-data repository. and has slightly different means of workflows depending on what type of release it is (alpha, beta, rc, patch release).  So the script that generates the new release tarballs will need to take some input to tell what sort of release it was and also be able to commit to the ansible-build-data repo.  I also found that I needed to make a special release today and so I ran a slightly different command to do that.  Special things might just be special, though.  It just means that we should make sure that it's easy for people to understand what the script does so that they can figure out how to do special, one-off things.  The build instructions are here: https://github.com/ansible-community/antsibull/blob/main/docs/build-ansible.rst

Reply:
I'm not sure I understand how it's bound to docs: can't this happen asynchronously? Also, building the docs can happen automatically if it's something the docs team is up to: an automatic workflow can send notifications to other systems, in particular, it can trigger workflows in other repos. Does the publish strictly depend on docs?